card/mod/05_standard/set/type/image.rb

```diff
@@ -12,10 +12,10 @@
 
   view :source do |args|
     style = case
-      when @mode==:closed ;  :icon
-      when args[:size]    ;  args[:size].to_sym
-      when main?          ;  :large
-      else                ;  :medium
+      when @mode==:closed then :icon
```

Comment:
Indent `when` as deep as `case`.<br>Surrounding space missing for operator `==`.